fastlane/lib/fastlane/version.rb

```diff
@@ -1,3 +1,3 @@
 module Fastlane
-  VERSION = '1.69.0'.freeze
+  VERSION = '1.70.0'.freeze
```

Comment:
Mh, does this release add anything new? We've had a fastlane release earlier today.